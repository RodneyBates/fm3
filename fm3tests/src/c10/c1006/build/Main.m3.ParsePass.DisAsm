 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L1(16_1),[1,0])
 ItkBlockDeclListLt(L4(16_4),[5,2])
 ItkSubtypeDeclLt([5,7])
 ItkDeclNo(D3,[5,7])
 ItkSubtypeDeclSubtype([5,9])
 ItkQualIdDeclNoAtom(D4,Id4"Y",[5,12],[5,16])
 ItkSubtypeDeclRt([5,7])
 ItkBlockDeclListSep(L1(16_1),[6,0])
 ItkSubtypeDeclLt([6,7])
 ItkDeclNo(D4,[6,7])
 ItkSubtypeDeclSubtype([6,9])
 ItkReservedId(Rid"<ROOT>",[6,12])
 ItkSubtypeDeclRt([6,7])
 ItkBlockDeclListSep(L2(16_2),[8,0])
 ItkFullRevealLt([8,9])
 ItkQualIdDeclNoAtom(D3,Id5"Q",[8,9],[8,14])
 ItkFullRevealEquals([8,16])
 ItkREFDefLt([8,18])
 ItkIdRefDeclNo(D4,[8,22])
 ItkREFDefRt([8,18])
 ItkFullRevealRt([8,9])
 ItkBlockDeclListSep(L3(16_3),[10,0])
 ItkPartialRevealLt([10,9])
 ItkIdRefAtom(Id6"R1",[10,9])
 ItkPartialRevealSubtype([11,2])
 ItkREFDefLt([11,5])
 ItkRecDefLt(L2(16_2),[11,9])
 ItkFieldDeclListLt(L2(16_2),[12,6])
 ItkFieldDeclLt([12,6])
 ItkFieldDeclIdListLt(L1(16_1),[12,6])
 ItkDeclNo(D1,[12,6])
 ItkFieldDeclIdListRt(L1(16_1),[12,6])
 ItkFieldDeclType([12,9])
 ItkIdRefDeclNo(D4,[12,11])
 ItkFieldDeclVal([13,4])
 ItkFieldDeclRt([12,6])
 ItkFieldDeclListSep(L1(16_1),[13,4])
 ItkFieldDeclLt([13,6])
 ItkFieldDeclIdListLt(L1(16_1),[13,6])
 ItkDeclNo(D2,[13,6])
 ItkFieldDeclIdListRt(L1(16_1),[13,6])
 ItkFieldDeclType([13,9])
 ItkIdRefDeclNo(D3,[13,11])
 ItkFieldDeclVal([14,4])
 ItkFieldDeclRt([13,6])
 ItkFieldDeclListRt(L2(16_2),[12,6])
 ItkRecDefRt(L2(16_2),[11,9])
 ItkREFDefRt([11,5])
 ItkPartialRevealRt([10,9])
 ItkBlockDeclListRt(L4(16_4),[5,2])
 ItkModuleBegin(L1(16_1),[16,2])
 ItkStmtListRt(L0(16_0),[17,2])
 ItkStmtListLt(L0(16_0),[17,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
