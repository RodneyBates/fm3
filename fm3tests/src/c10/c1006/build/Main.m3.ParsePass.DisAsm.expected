   275 16_00000000000000a2                   162 ItkEOF()
   273 16_00000000000000a3                   163 ItkLeftEnd()
   271 16_0000000000000167                   359 ItkExportIdListLt(L1(16_1),[1,7])
   266 16_000000000000016e                   366 ItkExportIdListElem(Id1"Main",[1,7])
   261 16_000000000000016a                   362 ItkExportIdListRt(L1(16_1),[1,7])
   256 16_00000000000000e6                   230 ItkBlockLt(L1(16_1),[5,2])
   251 16_00000000000001bf                   447 ItkBlockDeclListLt(L4(16_4),[5,2])
   246 16_00000000000000f6                   246 ItkSubtypeDeclLt([5,7])
   242 16_00000000000000b7                   183 ItkDeclNo(D3,[5,7])
   237 16_00000000000000fa                   250 ItkSubtypeDeclSubtype([5,9])
   233 16_00000000000000ba                   186 ItkQualIdAtoms(Id3"R",Id4"Y",[5,12],[5,16])
   225 16_00000000000000f9                   249 ItkSubtypeDeclRt([5,7])
   221 16_00000000000001c3                   451 ItkBlockDeclListSep(L1(16_1),[6,2])
   216 16_00000000000000f6                   246 ItkSubtypeDeclLt([6,7])
   212 16_00000000000000b7                   183 ItkDeclNo(D4,[6,7])
   207 16_00000000000000fa                   250 ItkSubtypeDeclSubtype([6,9])
   203 16_00000000000000b8                   184 ItkIdRefAtom(Id0"<NoAtomIdent>",[8,0])
   198 16_00000000000000f9                   249 ItkSubtypeDeclRt([6,7])
   194 16_00000000000001c3                   451 ItkBlockDeclListSep(L2(16_2),[8,2])
   189 16_00000000000000fd                   253 ItkFullRevealLt([8,9])
   185 16_00000000000000ba                   186 ItkQualIdAtoms(Id2"P",Id5"Q",[8,9],[8,14])
   177 16_0000000000000101                   257 ItkFullRevealEquals([8,16])
   173 16_000000000000010b                   267 ItkREFDefLt([8,18])
   169 16_00000000000000b9                   185 ItkIdRefDeclNo(D4,[8,22])
   164 16_000000000000010e                   270 ItkREFDefRt([8,18])
   160 16_0000000000000100                   256 ItkFullRevealRt([8,9])
   156 16_00000000000001c3                   451 ItkBlockDeclListSep(L3(16_3),[10,2])
   151 16_0000000000000104                   260 ItkPartialRevealLt([10,9])
   147 16_00000000000000b8                   184 ItkIdRefAtom(Id6"R1",[10,9])
   142 16_0000000000000108                   264 ItkPartialRevealSubtype([11,2])
   138 16_000000000000010b                   267 ItkREFDefLt([11,5])
   134 16_000000000000010f                   271 ItkRecDefLt(L2(16_2),[11,9])
   129 16_000000000000011d                   285 ItkFieldDeclListLt(L2(16_2),[12,6])
   124 16_0000000000000135                   309 ItkFieldDeclLt([12,6])
   120 16_0000000000000197                   407 ItkFieldDeclIdListLt(L1(16_1),[12,6])
   115 16_000000000000019e                   414 ItkFieldDeclIdListElem(Id7"F1",[12,6])
   110 16_000000000000019a                   410 ItkFieldDeclIdListRt(L1(16_1),[12,6])
   105 16_0000000000000139                   313 ItkFieldDeclType([12,9])
   101 16_00000000000000b9                   185 ItkIdRefDeclNo(D4,[12,11])
    96 16_000000000000013c                   316 ItkFieldDeclVal([13,4])
    92 16_0000000000000138                   312 ItkFieldDeclRt([12,6])
    88 16_0000000000000121                   289 ItkFieldDeclListSep(L1(16_1),[13,4])
    83 16_0000000000000135                   309 ItkFieldDeclLt([13,6])
    79 16_0000000000000197                   407 ItkFieldDeclIdListLt(L1(16_1),[13,6])
    74 16_000000000000019e                   414 ItkFieldDeclIdListElem(Id8"F2",[13,6])
    69 16_000000000000019a                   410 ItkFieldDeclIdListRt(L1(16_1),[13,6])
    64 16_0000000000000139                   313 ItkFieldDeclType([13,9])
    60 16_00000000000000b9                   185 ItkIdRefDeclNo(D3,[13,11])
    55 16_000000000000013c                   316 ItkFieldDeclVal([14,4])
    51 16_0000000000000138                   312 ItkFieldDeclRt([13,6])
    47 16_0000000000000120                   288 ItkFieldDeclListRt(L2(16_2),[12,6])
    42 16_0000000000000112                   274 ItkRecDefRt(L2(16_2),[11,9])
    37 16_000000000000010e                   270 ItkREFDefRt([11,5])
    33 16_0000000000000107                   263 ItkPartialRevealRt([10,9])
    29 16_00000000000001c2                   450 ItkBlockDeclListRt(L4(16_4),[5,2])
    24 16_00000000000000ea                   234 ItkBlockBEGIN(L1(16_1),[16,2])
    19 16_00000000000001ca                   458 ItkStmtListRt(L0(16_0),[17,2])
    14 16_00000000000001c7                   455 ItkStmtListLt(L0(16_0),[17,2])
     9 16_00000000000000e9                   233 ItkBlockRt(L1(16_1),[5,2])
     4 16_00000000000000a4                   164 ItkRightEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
